web/src/components/MapView.tsx

```diff
@@ -88,9 +87,12 @@ const MapView: React.FunctionComponent<MapViewProps> = (props: MapViewProps): Re
       <ReactMapGL
         interactiveLayerIds={[layerStyle.id!]}
         {...viewport}
+        height='100%'
+        width='100%'
         onViewportChange={setViewport}
         mapStyle={mapConfig.styleJSON}
-        onClick={clickItem}>
+        onClick={clickItem}
```

Comment:
Mostly it is nicer to name it depending on what it does. e.g. selectFeature.

Reply:
used `onSelectFeature`